File: second_week/chao_node.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc, char  *argv[])
{
    ros::init(argc, argv, "chao_node");
    printf("Hello World!\n");

    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("kuai_shang_che",10);

    ros::Rate loop_rate(10);

    while (ros::ok())
    {
        printf("别投\n");
        std_msgs::String msg;
        msg.data = "国服马超，带飞";
        pub.publish(msg);
        loop_rate.sleep();

    }
    
    return 0;
}

File: chao_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String

if __name__ == "__main__":
    rospy.init_node("chao_node")
    rospy.logwarn("你好")

    pub = rospy.Publisher("kuai_shang_che_kai_hei_qun",String,queue_size=10)

    rate = rospy.Rate(10)

    while not rospy.is_shutdown():
        rospy.loginfo("我要开始刷屏了")
        msg = String()
        msg.data = "马超带飞"
        pub.publish(msg)
        rate.sleep()